gateway-service/src/main/java/org/zowe/apiml/gateway/ws/WebSocketProxyClientHandler.java

```diff
@@ -38,4 +38,9 @@ public void afterConnectionClosed(WebSocketSession session, CloseStatus status)
         log.debug("afterConnectionClosed(session={},status={})", session, status);
         webSocketServerSession.close(status);
     }
+
+    @Override
+    public void handleTransportError(WebSocketSession session, Throwable exception) throws Exception {
+        log.warn(String.format("WebSocket transport error in session %s", session.getId()), exception);
```

Comment:
```suggestion
        log.warn("WebSocket transport error in session {}: {}", session.getId(), exception.getMessage());
```

Do we really need the stacktrace?

Reply:
yeah, probably not